test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -100,9 +90,9 @@ export default class GutenbergEditorComponent extends AbstractEditorComponent {
 	}
 
 	async getTitle() {
-		return await this.driver
-			.findElement( By.css( '.editor-post-title__input' ) )
-			.getAttribute( 'value' );
+		return await driverHelper.getInputText(
+			this.driver.findElement( By.css( '.editor-post-title__input' ) )
```

Comment:
Oh, 🤦🏻 , the missing `await` was my mistake. I'm not sure why that worked though, I'll double-check.

Reply:
Figured it out. It's quite a TIL and interesting. 

The `findElement` method actually returns a [WebElementPromise](https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/index_exports_WebElementPromise.html) which is an `IThenable`. However, its interface is also very similar to WebElement's in that it also exposes all query methods that are accessible without having to wait for it to resolve! These methods in turn return a `Promise, but these are `awaited` inside `getInputText` already, so that's why it didn't fail. 

In sum, in this case, it will work fine if you don't `await` for the `WebElementPromise` to resolve and just pass it as is to `getInputText`, as it has the right shape, and `getInputText` correctly handles the promises from the methods it calls. _Can doesn't mean should_, though, and the fact it's a `Promise` that's not being awaited is confusing, so I'll push a fix shortly.

Reply:
Oh I thought the `IThenable` wrapper is used for the `driver` only! Nice! Thanks for digging into it 🙌